data/hails.txt

```diff
@@ -1754,6 +1753,8 @@ phrase "friendly pirate"
 phrase "friendly pirate"
 	word
 		"You better watch "
+		"You best watch "
+		"You'd watch "
```

Comment:
anyone else thinks this line sounds bad? (example: "You'd watch your cargo around me!")

Reply:
I also do not agree with it.

Reply:
just changing it to "You should " seems like a fine fix

Reply:
I think *this* line is bad, but in the other two lines above, those should be replaced with "You'd" instead of "You".

I think a `word` containing solely a space should only be used if the one next to it contains a space in four (since the word "word" is four characters long) different statements. That is, in this situation for example, the current form is better because it's one character shorter overall, but if we were to add a fourth statement to it, *then* a separate space `word` could be used.

Reply:
I think "You better watch" and "You best watch" are exactly the kind of poor grammar that a pirate would use. :)

Reply:
It's not just a matter of grammatical incorrectness, it's also a matter of whether or not it is actually easier to say. I think "you'd better" is better with that brief stop than the smooth transition in "you better".